### ISSUE TITLE
zone attribute

### ISSUE THREAD

**Issue by JDWoodcock** [npct/pct-shiny#36]
hi 
i  like the comments but for zone attribute it is too wordy and I think includes a superfluous word- "Alters to colours", should it just say "Alters colours"?

**Comment by usr110** [comment]
I have reduced text to some extent. When @Robinlovelace or @nikolai-b will commit the latest code on the server, then you'd be able to see it.
As for your comment about this specific comment, I already have changed it to 'Set' - see the attached picture

![revised-tooltip](https://cloud.githubusercontent.com/assets/10958628/7415989/430f41f6-ef54-11e4-84ad-2c24dc29da5d.png)
